src/main/java/com/rackspacecloud/metrics/queryservice/providers/DevTestTenantRouteProvider.java

```diff
@@ -86,8 +88,6 @@ private TenantRoutes getStubbedRoutes(String tenantId, String measurement) {
 
     @Override
     public Collection<String> getMeasurements(String tenantId, RestTemplate restTemplate) {
-        String requestUrl = String.format("%s/%s/measurements", tenantRoutingServiceUrl, tenantId);
-
-        return restTemplate.getForObject(requestUrl, Collection.class);
+        return ImmutableList.of("devMeasurement1", "devMeasurement2", "devMeasurement3");
```

Comment:
If Ceres is on Java 11, can use `List.of`. (We're enjoying that one all over the Salus code.)